projects/quickjs/fuzz_regexp.c

```diff
@@ -0,0 +1,79 @@
+/* Copyright 2020 Google Inc.
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#include "libregexp.h"
+#include "quickjs-libc.h"
+
+#include <stdint.h>
+#include <stdio.h>
+
+#define CAPTURE_COUNT_MAX 255
+
+FILE *outfile=NULL;
```

Comment:
is this and `fprintf` calls below necessary?

Reply:
Well, I want to make sure the access to `capture[i]` is used so as to not get optimized away 
(like when you compile https://github.com/google/fuzzing/blob/master/tutorial/libFuzzer/fuzz_me.cc with -O1)
This is based on the example from quickjs in libregexp.c